### PR TITLE
fetch unknown persons to generate url

### PR DESCRIPTION
**Pull request by SuperTux88**

### CODE REVIEW

File: spec/federation_callbacks_spec.rb [hunk 1]
```diff
@@ -428,6 +428,26 @@
...
+
+      expect(
+        DiasporaFederation.callbacks.trigger(:fetch_person_url_to, person.diaspora_handle, "/path/on/pod")
+      ).to eq("https://#{pod.host}/path/on/pod")
+    end
+
+    it "forwards the DiscoveryError" do
+      diaspora_id = FactoryGirl.generate(:diaspora_id)
+      expect(Person).to receive(:find_or_fetch_by_identifier).with(diaspora_id)
+                          .and_raise(DiasporaFederation::Discovery::DiscoveryError)
```
Comment (diaspora-code-review) [inline spec/federation_callbacks_spec.rb]: Use 2 (not 20) spaces for indenting an expression spanning multiple lines.

**Comment by Flaburgan** [comment]
Does that help for https://github.com/diaspora/diaspora/issues/4491 ?

**Comment by jhass** [comment]
No.

**Comment by SuperTux88** [comment]
this fixes "Failed to fetch Post:&lt;guid&gt; from &lt;diaspora-id&gt; : NoMethodError: undefined method 'url_to' for nil:NilClass"

**Comment by denschub** [comment]
Dank je wel!

**Pull request merged**